erlang/potatoserver/rebar.config

```diff
@@ -0,0 +1,8 @@
+{erl_opts, [debug_info]}.
+{deps, []}.
+{plugins, [rebar3_hex]}.
```

Comment:
what is rebar3_hex?

Reply:
idk D: 

Wanna switch back to stack + haskell? ʘ‿ʘ